GermanFilter/sections/css_extended.txt

```diff
@@ -1,6 +1,9 @@
 !
 ! Extended CSS rules
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/83261
+ga.de#?#div.park-section:has( aside[data-portal-id^="mobilebanner_"])
```

Comment:
Blocks too much. For example
![image](https://user-images.githubusercontent.com/8361299/119346357-c9206780-bca2-11eb-9ce2-17e1075bdf11.png)

Reply:
@Alex-302 Does this work?
`ga.de#?#div.park-section:has( li.park-teaser-collection__list-item > aside[data-portal-id^="mobilebanner_"])`

Reply:
It seems it works.